Makefile.am

```diff
@@ -171,6 +171,8 @@ ci: rc-release
 	$(MAKE) TMPDIR=/var/tmp COVERAGE_PROCESS_START=$(abs_builddir)/.coveragerc check
 	@sudo $(MAKE) TMPDIR=/var/tmp COVERAGE_PROCESS_START=$(abs_builddir)/.coveragerc \
 		TESTS=nosetests_root.sh check
+	@sudo $(MAKE) TMPDIR=/var/tmp COVERAGE_PROCESS_START=$(abs_builddir)/.coveragerc \
+		TESTS=storage/run_storage_tests.py check
```

Comment:
> How come ? What about line 173 then ?

oh geez. nevermind. I was reading it as an argument to storage/run_storage_tests.py, sorry about that.